src/components/currentLesson/CurrentLesson.tsx

```diff
@@ -0,0 +1,52 @@
+import './_currentLesson.scss';
+import { useEffect, useState } from "react"
+import { CurrentLessonProps } from "../../types/components/componentType"
+import defaultCoursePhoto from '../../img/defaultCoursePhoto.png';
+import axios from "axios";
+
+const CurrentLesson = () => {
+const [currentLessonProps, setCurrentLessonProps] = useState<CurrentLessonProps | null>(null);
```

Comment:
Don't use word "Props" here. Because props - values that you pass from one component to other.

Reply:
fixed